File: readme/cn/selfhost/SDK/webfont-SDK-java(for1.22.2)/cn/webfont/client/model/WebFontInfo.java
package cn.webfont.client.model;

import lombok.Data;
import lombok.ToString;

import java.io.Serializable;
import java.math.BigInteger;

@Data
@ToString(exclude = {"bytesTtf", "bytesEot", "bytesWoff","bytesWoff2"})
public class WebFontInfo implements Serializable {

  private static final long serialVersionUID = 7974743939894430021L;
  private int fontId;
  private int fontFormatVersion;
  private long fontCheckSum;
  private long date;
  private String nameEn;

  private byte[] bytesTtf;
  private byte[] bytesEot;
  private byte[] bytesWoff;
  private byte[] bytesWoff2;
  private String content;
  private String apiKey;

  private String ttfUrl;
  private String woffUrl;
  private String eotUrl;
  private String woff2Url;
}

File: readme/cn/selfhost/SDK/webfont-SDK-NetCore(for1.22.2)/WebFontClient.SDK/Models/WebFontInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WebFontClient.SDK.Models
{
    public class WebFontInfo
    {
        public const long serialVersionUID = 7974743939894430021L;
        public int fontId;
        public int fontFormatVersion;
        public long fontCheckSum;
        public long date;
        public string nameEn;
        public byte[] bytesTtf;
        public byte[] bytesEot;
        public byte[] bytesWoff;
        public byte[] bytesWoff2;
        public string content;
        public string apiKey;
        public string ttfUrl;
        public string woffUrl;
        public string woff2Url;
        public string eotUrl;
    }
}
